src/ex5/server/services/item_manager.js

```diff
@@ -1,53 +1,70 @@
-const PokemonClient = require('../clients/pokemon_client')
-
+const PokemonClient = require("../clients/pokemon_client")
+const { Items } = require("../db/models")
 class ItemManager {
-    constructor() {
-        this.pokemonClient = new PokemonClient();
-        this.items = []; //TODO: remove, items should be stored to DB using Item sequelize model
-    }
+  constructor() {
+    this.pokemonClient = new PokemonClient()
+  }
 
-    getItems = () => this.items
+  getItems = async () => {
+    const items = await Items.findAll()
+    return items
+  }
 
-    handleItem = async item => {
-        if (this._isNumber(item)) { return await this.fetchAndAddPokemon(item); }
-        if (this._isList(item)) { return await this.fetchAndAddManyPokemon(item); }
-
-        this.addItem(item)
+  handleItem = async (item) => {
+    if (this._isNumber(item)) {
+      return await this.fetchAndAddPokemon(item)
     }
-
-    addItem = item => {
-        this.items.push(item);
+    if (this._isList(item)) {
+      return await this.fetchAndAddPokemons(item)
     }
 
-    addPokemonItem = pokemon => {
-        this.addItem(`Catch ${pokemon.name}`);
-    }
+    this.addItem(item)
+  }
 
-    fetchAndAddPokemon = async pokemonId => {
-        try {
-            const pokemon = await this.pokemonClient.getPokemon(pokemonId);
-            this.addPokemonItem(pokemon);
-        } catch (error) {
-            this.addItem(`Pokemon with ID ${pokemonId} was not found`);
-        }
-    }
+  addItem = (item) => {
+    Items.create({
+      ItemName: item,
+    })
+  }
 
-    fetchAndAddManyPokemon = async inputValue => {
-        try {
-            const pokemons = await this.pokemonClient.getManyPokemon(inputValue.replace("/ /g", "").split(","));
-            pokemons.forEach(this.addPokemonItem);
-        } catch (error) {
-            console.error(error)
-            this.addItem(`Failed to fetch pokemon with this input: ${inputValue}`)
-        }
+  addPokemonItem = (pokemon) => {
+    Items.create({
+      ItemName: `Catch ${pokemon.name}`,
+      id: pokemon.id,
+    })
```

Comment:
do you really want the item id to be the pokemon id? 
what would happen if I have regular item with id 1 and then I'll add pokemon id 1?